lib/splitclient-rb/cache/repositories/events_repository.rb

```diff
@@ -25,15 +33,23 @@ def metadata
           }
         end
 
-        def event(key, traffic_type, event_type, time, value)
+        def event(key, traffic_type, event_type, time, value, properties, size = nil)
```

Comment:
Can we not pollute the Event DTO with the size? As an alternative, on other languages we used a wrapper for this.

Reply:
Removed